types/moment-range/package.json

```diff
@@ -1,5 +1,6 @@
 {
     "dependencies": {
-        "moment": ">=2.14.0"
+        "moment": "2.17.1",
+        "moment-range": "3.0.3"
```

Comment:
Actually, the dependency on `moment-range` isn't necessary. `package.json` is just for dependencies on types, but if `moment-range` provided its own types, we wouldn't have it on DefinitelyTyped in the first place.